docs/abi-spec.rst

```diff
@@ -684,6 +684,8 @@ Through ``abi.encodePacked()``, Solidity supports a non-standard packed mode whe
 
 Furthermore, structs as well as nested arrays are not supported.
 
+Note that the encoding of fixed-size types returns bytes the size of the sum of the inputs, e.g. encoding two addresses (20 bytes each) returns a byte string of length 40. 
```

Comment:
What is the difference between this and `types shorter than 32 bytes are neither zero padded nor sign extended`?

Reply:
From reading I hadn't realized that the values would simply be concatenated. But now I agree `types shorter than 32 bytes are neither zero padded nor sign extended` covers it. Feel free to close this pr

Reply:
So maybe can you change the above to "types shorter than 32 bytes are concatenated directly, without padding or sign extension"?

Reply:
I agree that's more clear. Just updated this diff